src/ProjectService.Models.Db/DbTaskProperty.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.ProjectService.Models.Db
+{
+    public class DbTaskProperty
+    {
+        public const string TableName = "TaskProperties";
+
+        public Guid Id { get; set; }
+        public string Name { get; set; }
+        public Guid ProjectId { get; set; }
+        public Guid AuthorId { get; set; }
+        public int PropertyType { get; set; }
+        public string Description { get; set; }
+        public DateTime CreatedAt { get; set; }
+        public bool IsActive { get; set; }
+
+        public DbProject Project { get; set; }
```

Comment:
you have 5 classes in your class, but in the communication configuration only for 2

Reply:
It doesn't depend on classes, as I understand, you can see the same thing in DbProject in ProjectService model Db

Reply:
you can create link without foreight key